demo/src/right-resizable-box.js

```diff
@@ -66,27 +68,26 @@ const RightResizableBox = (props) => {
     };
 
     return (
-        <ResizableBox
+        <ResizableBoxSx
             style={{ display: props.hide ? 'none' : undefined }}
             width={
                 props.fullscreen
                     ? windowWidth
                     : windowWidth * resizedTreePercentage
             }
-            className={clsx(classes.panel, {
-                [classes.resizePanelHandle]: !props.disableResize,
-            })}
+            sx={mergeSx(
```

Comment:
cf doc : https://www.npmjs.com/package/react-resizable?activeTab=readme
it says : "Unknown props that are not in the list below will be passed to the child component."
nor `className` nor `sx` are a known prop, maybe set it directly to the Box child ?
Or I miss something ?

Reply:
sx is handled by styled and transformed to a classname prop on the component.
Not sure what's best between setting them on the resizable or the box